src/processTargets.ts

```diff
@@ -263,16 +263,16 @@ function transformSelection(
       return [{ selection, context: {} }];
 
     case "containingScope":
-      let node: SyntaxNode | null = context.getNodeAtLocation(
-        new Location(selection.editor.document.uri, selection.selection)
-      );
-
       const nodeMatcher = getNodeMatcher(
         selection.editor.document.languageId,
         modifier.scopeType,
         modifier.includeSiblings ?? false
       );
 
+      let node: SyntaxNode | null = context.getNodeAtLocation(
```

Comment:
moved this one so that cursorless won't try to access syntax tree unless it supports the given language